test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -267,82 +270,470 @@ var _ = g.Describe("Scylla Manager integration", func() {
+		o.Expect(*sourceSC.Status.Backups[0].Error).NotTo(o.BeEmpty())
+		o.Expect(sourceSC.Status.Repairs).To(o.BeEmpty())
+
+		previousBackupTask := backupTask
+		framework.By("Verifying that backup task in manager state wasn't modified")
+		tasks, err = managerClient.ListTasks(ctx, *sourceSC.Status.ManagerID, "backup", false, "", "")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(tasks.TaskListItemSlice).To(o.HaveLen(1))
+		backupTask = tasks.TaskListItemSlice[0]
+		o.Expect(backupTask.Name).To(o.Equal(sourceSC.Status.Backups[0].Name))
+		o.Expect(backupTask.ID).To(o.Equal(*sourceSC.Status.Backups[0].ID))
+		o.Expect(backupTask.Properties).To(o.Equal(previousBackupTask.Properties))
+
+		framework.By("Deleting the backup task for source ScyllaCluster")
+		sourceSC, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Patch(
+			ctx,
+			sourceSC.Name,
+			types.JSONPatchType,
+			[]byte(`[{"op":"remove","path":"/spec/backups/0"}]`),
+			metav1.PatchOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(sourceSC.Spec.Backups).To(o.BeEmpty())
+
+		framework.By("Waiting for source ScyllaCluster to sync backup task deletion with Scylla Manager")
+		backupTaskDeletedCond := func(cluster *scyllav1.ScyllaCluster) (bool, error) {
+			return len(cluster.Status.Backups) == 0, nil
+		}
+
+		waitCtx6, waitCtx6Cancel := utils.ContextForManagerSync(ctx, sourceSC)
+		defer waitCtx6Cancel()
+		sourceSC, err = controllerhelpers.WaitForScyllaClusterState(waitCtx6, f.ScyllaClient().ScyllaV1().ScyllaClusters(sourceSC.Namespace), sourceSC.Name, controllerhelpers.WaitForStateOptions{}, backupTaskDeletedCond)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Verifying that backup task deletion was synchronized")
+		tasks, err = managerClient.ListTasks(ctx, *sourceSC.Status.ManagerID, "backup", false, "", "")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(tasks.TaskListItemSlice).To(o.BeEmpty())
+
+		// Close the existing session to avoid polluting the logs.
+		di.Close()
+
+		framework.By("Deleting source ScyllaCluster")
+		var propagationPolicy = metav1.DeletePropagationForeground
+		err = f.ScyllaClient().ScyllaV1().ScyllaClusters(sourceSC.Namespace).Delete(ctx, sourceSC.Name, metav1.DeleteOptions{
+			Preconditions: &metav1.Preconditions{
+				UID: &sourceSC.UID,
+			},
+			PropagationPolicy: &propagationPolicy,
+		})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		waitCtx7, waitCtx7Cancel := context.WithCancel(ctx)
+		defer waitCtx7Cancel()
+		err = framework.WaitForObjectDeletion(
+			waitCtx7,
+			f.DynamicClient(),
+			scyllav1.GroupVersion.WithResource("scyllaclusters"),
+			sourceSC.Namespace,
+			sourceSC.Name,
+			&sourceSC.UID,
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		targetSC := f.GetDefaultScyllaCluster()
+		targetSC.Spec.Datacenter.Racks[0].Members = sourceSC.Spec.Datacenter.Racks[0].Members
+		targetSC.Spec.ScyllaArgs = "--consistent-cluster-management=false"
```

Comment:
Add comment about why it's needed

Reply:
done

Reply:
Is there an issue in Scylla which we can reference to? I guess there will be a schema restore solution for clusters having this option enabled.

Reply:
> Is there an issue in Scylla which we can reference to?

Not that I know of, no. The workaround for 5.4.x will stay. 

> I guess there will be a schema restore solution for clusters having this option enabled.

It's not possible currently, you need a fresh cluster with `consistent_cluster_management: false`. 6.x.x is just going to have a different set of workaround afaik.